modules/nw-dns-operator-managementState.adoc

```diff
@@ -6,19 +6,37 @@
 [id="nw-dns-operator-managementState_{context}"]
 = Changing the DNS Operator managementState
 
-DNS manages the CoreDNS component to provide a name resolution service for pods and services in the cluster. The `managementState` of the DNS Operator is set to `Managed` by default, which means that the DNS Operator is actively managing its resources. You can change it to `Unmanaged`, which means the DNS Operator is not managing its resources.
+The DNS Operator manages the CoreDNS component to provide a name resolution service for pods and services in the cluster. The `managementState` of the DNS Operator is set to `Managed` by default, which means that the DNS Operator is actively managing its resources. You can change it to `Unmanaged`, which means the DNS Operator is not managing its resources.
 
 The following are use cases for changing the DNS Operator `managementState`:
 
-* You are a developer and want to test a configuration change to see if it fixes an issue in CoreDNS. You can stop the DNS Operator from overwriting the fix by setting the `managementState` to `Unmanaged`.
+* You are a developer and want to test a configuration change to see if it fixes an issue in CoreDNS. You can stop the DNS Operator from overwriting the configuration change by setting the `managementState` to `Unmanaged`.
 
 * You are a cluster administrator and have reported an issue with CoreDNS, but need to apply a workaround until the issue is fixed. You can set the `managementState` field of the DNS Operator to `Unmanaged` to apply the workaround.
 
 .Procedure
 
-* Change `managementState` DNS Operator:
+. Change `managementState` to `Unmanaged` in the DNS Operator:
 +
 [source,terminal]
 ----
 oc patch dns.operator.openshift.io default --type merge --patch '{"spec":{"managementState":"Unmanaged"}}'
 ----
+
+. Review `managementState` of the DNS Operator using the `jq` command line JSON parser:
```

Comment:
I saw this discussed recently - not sure if this is applicable to these docs or not, but I'll mention it just in case:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#commands-with-jq

Reply:
@candita It looks like we avoid jq commands, when possible. Can you please suggest an alternative? Thanks!

Reply:
Thank you! All updated.